Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -231,6 +231,11 @@
       - GrenadeFlashBang # Goobstation
       - BolaEnergy # Goobstation
       - CartridgeEpinephrine # Goobstation
+      - CartridgeBicaridine # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)